model.py

```diff
@@ -3250,6 +3250,8 @@ def set_summary(self, resource):
         # TODO: clean up the content
         if resource:
```

Comment:
maybe I should make this `if resource and resource.representation`, so I can set a description with an empty string instead of setting the resource to None?